saiku-webapp/src/main/resources/ccrs-mondrian-cp1.xml

```diff
@@ -248,6 +248,46 @@
 		</Rows>
 	</InlineTable>
 
+        <Query alias="CP1OMTMember">
+            <ExpressionView>
+                <SQL dialect="generic">
+                    <![CDATA[
+                        select
+                            M.ID, M.PARENT_ID, M.AGENCY_ID,
+                            CASE WHEN M.NAME IS NULL THEN '' ELSE M.NAME END as NAME,
+                            CASE WHEN M.EMAIL IS NULL THEN '' ELSE M.EMAIL END as EMAIL,
+                            CP1.DESIGNATION, CP1.ENTITY_YEAR, CP1.COUNTRY_INCOME_STATUS_ID, CP1.SECONDARY_COUNTRY_INCOME_STATUS_ID,
+                            CASE WHEN cp1.OFFICIAL_REQUEST_DAO_COUNTRY = TRUE THEN 1 ELSE 0 END as OFFICIAL_REQUEST_DAO_COUNTRY
+                        from
+                            CP1OMTMEMBER M, CP1GENERAL_PROFILE CP1, BASE_COUNTRY bc, V_CP_VALIDATED_DESIGNATIONS v
+                        WHERE
+                            M.PARENT_ID = CP1.ID AND
+                            CP1.DESIGNATION != 'AQ' AND CP1.DESIGNATION != 'DOCO' AND CP1.DESIGNATION = bc.ISO2 AND
+                            CP1.DESIGNATION = v.DESIGNATION AND v.ENTITY_YEAR = CP1.ENTITY_YEAR
+                    ]]>
+                </SQL>
+            </ExpressionView>
+        </Query>
+        <InlineTable alias="DAOCP1OMTMember">
```

Comment:
It seems like this inline table serves no purpose.

Reply:
I fixed the usage of the table. Good catch!